Product page

Jayesh3636 opened the pull request:
Added Product page!
Review it and let me know the changes
TIA~!

Jayesh3636 committed:
v1.00 Added full product page

Jayesh3636 committed:
v1.01 Fixes: search menu margins

pavan6115 commented on productPage/productPage.css:
```diff
@@ -0,0 +1,242 @@
+/* * {
+    font-family: 'Montserrat', 'sans-serif';
+} */
+
```
Remove comments before pushing
UI looks clean 👍

Jayesh3636 replied:
Done!

khushijohri2001 commented on productPage/productPage.css:
```diff
@@ -0,0 +1,242 @@
+
+.header {
+    grid-area: navbar;
+    /* border: 2px solid red; */
```
You must remove these comments if not needed now. Unwanted comments make your code looks messy

Jayesh3636 replied:
done

khushijohri2001 commented on productPage/productPage.css:
```diff
@@ -0,0 +1,242 @@
+    top: 21.4rem;
+    border-radius: 1.2rem;
+    border: none;
+    background: #2780F2;
```
you can use a var for color and mention it in the root, then you the variables instead of direct color code.

Jayesh3636 replied:
done!

khushijohri2001 commented on productPage/productPage.css:
```diff
@@ -0,0 +1,242 @@
+    border-radius: 1.2rem;
+    border: none;
+    background: #2780F2;
+    font-family: 'Karla';
```
The same applies to font-family too.

Jayesh3636 replied:
done~!

khushijohri2001 commented on productPage/productPage.css:
```diff
@@ -0,0 +1,242 @@
+}
+
+.search-bar {
+    height: 55px;
```
try to avoid the use of px unit after 10px

Jayesh3636 replied:
done~!

khushijohri2001 commented on productPage/productPage.html:
```diff
@@ -0,0 +1,616 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
```
Not mandatory but you can also use Meta tags for author and description here for best practice

Jayesh3636 replied:
not so used to it but i ll try

khushijohri2001 reviewed (approved):
Just try to remove extra spaces otherwise the code is well aligned. The naming consistency is nice.

SukanyaSen530 reviewed (approved):
Liked the cool tones of colors you chose :)

Jayesh3636 committed:
v1.02 Fixes: Structuring and linking

Added new  file structure and added links to landing page.

also the review changes have been made

Jayesh3636 commented:
Thank you for reviewing it ! Guys merging it now!
